CONTRIBUTING.md

```diff
@@ -40,29 +40,58 @@ If the commit reverts a previous commit, it should begin with `revert: `, follow
 * **docs**: Documentation only changes
-* **feat**: A new feature
-* **fix**: A bug fix
-* **perf**: A code change that improves performance
-* **refactor**: A code change that neither fixes a bug nor adds a feature
-* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-* **test**: Adding missing tests or correcting existing tests
+* **chore**: Changes that do not affect the meaning of the code (formatting/linting, etc). Changes as a part of routine tasks like **Pre/Post release operation** (version bump + CHANGELOG, merge fix, etc).
+* **test(s)**: Adding missing tests or correcting existing tests
 
 ### Subject
-The subject contains succinct description of the change:
+The subject contains a succinct description of the change:
 
 * use the imperative, present tense: "change" not "changed" nor "changes"
 * don't capitalize first letter
 * no dot (.) at the end
+* an issue reference e.g. SDK-1234 can be added to the end of the subject in parentheses
 
 ### Body
 Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
 The body should include the motivation for the change and contrast this with previous behavior.
 
 ### Footer
-The footer should contain any information about **Breaking Changes** and is also the place to
-reference GitHub issues that this commit **Closes**.
-
-**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
\ No newline at end of file
+The footer should reference GitHub issues that this commit **Closes**.
+
+### Examples
+#### Bad examples:
+* _**Pull request #519:**_ improve: refactor user data handling (SDKCF-3970)
+<br>Remove default prefixes added by Bitbucket.
+ 
+* improve: refactor user data handling (SDKCF-3970) _**(#23)**_
+<br>Remove default suffixes added by Github.
+ 
+* _**Revert "refactor user data handling"**_
+<br>Revert should start with `revert:` followed by exact original commit message of reverted commit (without quotes).
+ 
+* fix: _**fix crash on iOS 15**_ (SDKCF-0000)
+<br>Commit message should contain some context. In this case - a cause of the crash.
+ 
+* feat: _**add new build configuration for users that require user data to be stored in another directory and add migration logic with exception of users migration from SDK version 1.0.0**_
+<br>Try to fit within 100 characters limit. Use abbreviations if necessary. Focus on the most important change.
+
+#### Good examples:
+* improve: refactor user data handling (SDKCF-3970)
+* improve: refine URL session configuration, add timeouts (SDKCF-0000)
+* docs: update README with SPM information (SDKCF-4388)
+* chore: bump RSDKUtils to 2.1.0 (SDKCF-4392)
+* chore: prepare 5.0.0 release (SDKCF-4160)
+* revert: improve: refine URL session configuration, add timeouts (SDKCF-0000)
```

Comment:
Can we use two <**type**> together?

Reply:
Yes. That's an exception for revert commits. It starts with `revert:` followed by exact message of reverted commit. Maybe we could put the reverted commit message in "", I'm not sure